docs/3.x.x/en/plugin-development/utils.md

```diff
@@ -2,6 +2,63 @@
 
 Strapi provides helpers so you don't have to develop again and again the same generic functions.
 
+## Auth
+
+`auth.js` lets you get, set and delete data in either the browser's `localStorage` or `sessionStorage`.
+
+### Methods
+
+| Name | Description |
+| ---- | ––––––––––– |
+| clear(key) | Remove the data in either `localStorage` or `sessionStorage` |
+| clearAppStorage() | Remove all data from both storage |
+| clearToken() | Remove the user's `jwt Token` in the appropriate browser's storage |
+| clearUserInfo() | Remove the user's info from storage |
+| get(key) | Get the item in the browser's storage |
```

Comment:
Is it from the `sessionStorage` or from the `localStorage`?

Reply:
Add `isLocalStorage` as parameter in this function.

Reply:
There's no parameter needed in this function. Here's its code :

```
get(key) {
    if (localStorage && localStorage.getItem(key)) {
      return parse(localStorage.getItem(key));
    }

    if (sessionStorage && sessionStorage.getItem(key)) {
      return parse(sessionStorage.getItem(key));
    }

    return null;
  },
```